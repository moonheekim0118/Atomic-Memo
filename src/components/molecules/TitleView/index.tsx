import React from 'react';
import Span from '../../atoms/Span';
import styled from 'styled-components';

interface Props {
  text: string;
  time: string;
}

const TitleView = ({ text, time }: Props) => {
  return (
    <StyledTitleView>
      <Span size="title">{text}</Span>
      <Span size="small">{time}</Span>
    </StyledTitleView>
  );
};

const StyledTitleView = styled.div`
  padding: 2em;
  display: flex;
  flex-direction: column;
`;
export default TitleView;
