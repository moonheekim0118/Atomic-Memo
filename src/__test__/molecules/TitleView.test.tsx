import 'jsdom-global/register';
import React from 'react';
import TitleView from '../../components/molecules/TitleView';
import { mount } from 'enzyme';

describe('<TitleView/>', () => {
  let container = null;
  it('renders correctly', () => {
    container = mount(<TitleView text="test" time="test" />);
  });

  it('matches snpashot', () => {
    expect(container.html()).toMatchSnapshot();
  });
});
