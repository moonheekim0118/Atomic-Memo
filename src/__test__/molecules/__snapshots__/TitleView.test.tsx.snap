// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleView/> matches snpashot 1`] = `"<div class=\\"TitleView__StyledTitleView-sc-18c7sy9-0 bPlOFX\\"><span width=\\"auto\\" display=\\"inline\\" class=\\"Span__StyledSpan-sc-10y5m6u-0 jaanRG title\\">test</span><span width=\\"auto\\" display=\\"inline\\" class=\\"Span__StyledSpan-sc-10y5m6u-0 jaanRG small\\">test</span></div>"`;
